docs/tutorial/ordering.md

```diff
@@ -0,0 +1,83 @@
+# **Ordering**
+
+**Django Ninja Extra** provides an intuitive ordering model using `ordering` decoration from the Django-Ninja-Extra ordering module. It expects a Queryset from as a route function result.
+
+> This feature was inspired by the [DRF OrderingFilter][drf-ordering]
```

Comment:
I think you need to add the url for this here `[DRF OrderingFilter](URL)`

Reply:
np